src/Symfony/Component/PropertyAccess/Annotation/Property.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Property
```

Comment:
This annotation duplicates the other annotations. I don't think we should add multiple ways to add the same thing:

* more code to maintain
* harder to improve performance
* harder to introduce changes consistently
* harder to understand for our users when to use which